# stopped task
string commanded_task
